staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1264,21 +1272,21 @@ func (c *Cacher) getBookmarkAfterResourceVersionLockedFunc(ctx context.Context,
 //   - start at Any (return the current watchCache's RV)
 //
 // The returned function must be called under the watchCache lock.
-func (c *Cacher) getStartResourceVersionForWatchLockedFunc(ctx context.Context, parsedWatchResourceVersion uint64, opts storage.ListOptions) (func() uint64, error) {
```

Comment:
I expected context to be preserved.  Even if not used now, I like having it plumbed